app/src/pages/inside/testPlansPage/milestonesTable/milestoneCard.scss

```
@@ -0,0 +1,200 @@
+  &__status-chevron {
+    display: inline-flex;
+    flex-shrink: 0;
+    line-height: 0;
+    color: var(--rp-ui-base-bg-000);
+    transition: transform 0.2s ease;
+
+    &_open {
+      transform: rotate(180deg);
+    }
+
+    :global(svg) {
```

Stylelint (17.4.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)